Installfest (#1)
Ada-Developers-Academy/ada-precourse-v2: 00-preparing-for-ada/00-starter.md

<!-- available callout types: info, success, warning, danger, secondary  -->
### !callout-info

## You can resubmit

In this course you can always resubmit work.  If you want a 2nd-round of instructor feedback let us know on Slack as we normally only give feedback once.

### !end-callout

<!-- available callout types: info, success, warning, danger, secondary  -->
### !callout-danger

## Due Dates

# TODO - Add Due Dates

### !end-callout

## Pre-Ada Units

These units are designed with units to match the 6-day in-person Pre-Ada sessions for students determined to need a bit of additional support called Precourse Live.  If you want to attend Live sessions, you can request to do so, but space is limited and admission is based on need.

## Learning Goals prior to starting at Ada

In this course we want you to learn to do the following:

*  Consistently follow [Python coding practices](https://www.python.org/dev/peps/pep-0008/)

> *  Understand the of order of operations reflected in use of relational and logical operators
> *  Understand and use complex conditional statements
> *  Understand and use iterators and loops
> *  Understand and use lists along with loops
> *  Understand and use dictionaries along with loops
> *  Understand and use lists and dictionaries together - nested data structures with loops
> *  Learn HTML & CSS

Reviewer: ```suggestion
*  Understand and use conditional statements
*  Understand and use functions
*  Understand and use loops
*  Understand and use lists along with loops
*  Understand and use dictionaries
*  Understand and use lists and dictionaries together - nested data structures with loops
```

We should check w/ the new curriculum to figure out how much of the conditional and order of ops content the pre-course should cover, from what I've read I think some of this is going to be taught in the first week so we may not need the depth that is suggested here.

Re: iterators - Iterators are a big topic, we're going to touch on them lightly in the context of looping over a list, but I think not with enough depth to list it here.  Also, re: HTML & CSS - that's not in the current plan for the course, do we need to add it?

Author: We'll have them go through a brief and free Udemy course for HTML/CSS, just to get an introduction.
